content/account/sso.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Single sign-on (SSO)
+meta_description: "Single sign-on enables users to authenticate with Ably using your own identity provider."
+meta_keywords: "sso, saml, strict mode, single sign-on"
+---
+
+Single sign-on (SSO) enables your users to authenticate via any SAML-compatible identity provider.
+
+<aside date-type='note'>
+<p>User provisioning and deprovisioning via SCIM or SAML JIT is not supported.</p>
+</aside>
+
+h2(#configure). Configure
+
+Single sign-on is restricted to Enterprise customers only and must be enabled on a per-account basis by "contacting Ably":https://ably.com/contact. Only "account owners":/account/users can configure SSO for an account.
+
+Any SAML-compatible identity provider can be used to enable SSO.
+
+The following instructions are examples of configuring SSO with "Okta":#okta and "Google Workspace":#google.
+
+h3(#okta). Okta
+
+To enable SSO using "Okta":https://www.okta.com/ as the identity provider:
+
+In your Ably account dashboard:
+
+1. Log in to your "account":https://ably.com/accounts/any.
+2. Select *Settings* from the account navigation dropdown.
+3. Toggle *Enable Single Sign-On* under the *Authentication Settings* section.
+4. Note down the *Single sign-on URL* and *Audience URI* values.
+
+In your Okta account:
+
+Use the "Okta guide":https://developer.okta.com/docs/guides/saml-application-setup/overview/ for enabling SSO.
+
+1. Upload the Ably logo.
+2. Select *EmailAddress* for the *Name ID format* field.
+3. Select *Email* for the *Application username* field.
+4. Ably requires users' full names. Okta can share user profile fields as SAML attributes.
+5. Ensure you assign users to the newly created Okta application.
+6. Note down the *Identity provider metadata* from Okta.
```

Comment:
The Okta docs use "Identity Provider metadata". Very small detail. Maybe not worth pursuing. 

```suggestion
6. Note down the *Identity Provider metadata* from Okta.
```

Reply:
Fixed, thanks!